modules/farm_fd2/src/entrypoints/transplanting/transplanting.submitReset.e2e.cy.js

```diff
@@ -0,0 +1,284 @@
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="selector-input"]')
+      .select('ALF');
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="picker-options"]')
+      .find('input')
+      .eq(0)
+      .click();
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="picker-options"]')
+      .find('input')
+      .eq(3)
+      .click();
+    cy.get('[data-cy="transplanting-location"]').find(
+      '[data-cy="selector-input"]'
+    );
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+  });
+
+  it('Invalid bed feet disables submit', () => {
+    populateForm();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+    cy.get('[data-cy="transplanting-bed-feet"]')
+      .find('[data-cy="numeric-input"]')
+      .clear();
+    cy.get('[data-cy="transplanting-bed-feet"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+
+    cy.get('[data-cy="submit-button"]').click();
+    cy.get('[data-cy="submit-button"]').should('be.disabled');
+    cy.get('[data-cy="transplanting-bed-feet"]')
+      .find('[data-cy="numeric-input"]')
+      .type(3);
+    cy.get('[data-cy="transplanting-bed-feet"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+  });
+
+  it('Invalid bed width disables submit', () => {
+    populateForm();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+    cy.get('[data-cy="transplanting-bed-width"]')
+      .find('[data-cy="numeric-input"]')
+      .clear();
+    cy.get('[data-cy="transplanting-bed-width"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+
+    cy.get('[data-cy="submit-button"]').click();
+    cy.get('[data-cy="submit-button"]').should('be.disabled');
+    cy.get('[data-cy="transplanting-bed-width"]')
+      .find('[data-cy="numeric-input"]')
+      .type(5);
+    cy.get('[data-cy="transplanting-bed-width"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+  });
+
+  // Note: There is no way to make Rows/Bed invalid.
+
+  it('Invalid depth disables submit if equipment is selected', () => {
+    populateForm();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+    cy.get('[data-cy="soil-disturbance-depth"]')
+      .find('[data-cy="numeric-input"]')
+      .clear();
+    cy.get('[data-cy="soil-disturbance-depth"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+
+    cy.get('[data-cy="submit-button"]').click();
+    cy.get('[data-cy="submit-button"]').should('be.disabled');
+
+    cy.get('[data-cy="soil-disturbance-depth"]')
+      .find('[data-cy="numeric-input"]')
+      .type(9);
+    cy.get('[data-cy="soil-disturbance-depth"]')
+      .find('[data-cy="numeric-input"]')
+      .blur();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
+
+    cy.get('[data-cy="equipment-selector-1"]')
```

Comment:
Let's create a separate test that does:
- enter equipment
- make depth invalid
- click submit
- check submit is disabled
- delete equipment
- check submit is enabled

Also add another separate test like this for the speed field.